projects/storefrontlib/src/cms-components/configurator/commons/config-form/config-form.component.ts

```diff
@@ -46,7 +44,7 @@ export class ConfigFormComponent implements OnInit {
       );
 
     this.currentGroup$ = this.configRouterExtractorService
-      .extractRouterData(this.routingService)
+      .extractRouterData()
```

Comment:
The `updateConfiguration` might be doing too much. We can try to refactor it a bit, and avoid the explicit subscription, if possible.

Reply:
Also, I just noticed the `isLoading.valueOf()`. 
The `isLoading` is already a primitive type, so there's no need to use `valueOf`. 
Can you please check if you have instances of this in some other place?